app/src/main/java/by/paranoidandroid/dailyvisualizer/view/fragments/DayFragment.java

```diff
@@ -104,9 +103,18 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
                 } else {
                     ivDay.setImageDrawable(null);
                 }
+                if(day.getLatitude() != null){
+                    btShowLocation.setVisibility(View.VISIBLE);
+                    btShowLocation.setOnClickListener(v->{
+                      LocationMapManager.showLocation(getActivity(), day.getLatitude(), day.getLongitude());
+                    });
+                } else {
+                    btShowLocation.setVisibility(View.INVISIBLE);
+                }
             } else {
                 tvDescription.setText(getString(R.string.label_empty_day));
                 ivDay.setImageDrawable(null);
+                btShowLocation.setVisibility(View.INVISIBLE);
```

Comment:
And here too (maybe is it better to change visibility from `View.INVISIBLE` to `View.GONE`?)